project/AppDependencies.scala

```diff
@@ -6,16 +6,15 @@ object AppDependencies {
   val compile = Seq(
     play.sbt.PlayImport.ws,
     "org.reactivemongo" %% "play2-reactivemongo"            % "0.20.3-play26",
```

Comment:
Cann't we use latest version of this library like `1.0.0-play26`?

Reply:
I will try. Not sure about the underlying implications. As this is also not using simple reactive mongo will see if it works